python/ray/_private/import_thread.py

```diff
@@ -156,9 +156,6 @@ def _process_key(self, key):
             # for profiling).
             # with profiling.profile("register_remote_function"):
             (self.worker.function_actor_manager.fetch_and_register_remote_function(key))
-        elif key.startswith(b"FunctionsToRun:"):
-            with profiling.profile("fetch_and_run_function"):
```

Comment:
Does it mean we are losing this data? This is kind of important for the timeline feature.